scss/_variables.scss

```diff
@@ -97,11 +97,6 @@ $cell-padding-y: 8px !default;
 $cell-padding: 8px !default;
 // $cell-padding-sm: $table-sm-cell-padding !default;
 
-$window-inner-padding-x: 16px !default;
-$window-inner-padding-y: 16px !default;
-$window-titlebar-padding-x: 16px !default;
-$window-titlebar-padding-y: 16px !default;
```

Comment:
Those variables are good here.  We first keep "layout" or "metrics" variables and then we put the "theme" or "skin" variables.